src/components/Game/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { FC } from "react";
+import styles from "./style.module.scss";
+import { useState } from "react";
+import GuessInput from "../GuessInput";
+import SubmitHighscore from "../SubmitHighscore";
+import { HighscoreObject } from "../../App";
+
+type GameProps = {
+  gameComplete: (highscore: HighscoreObject) => void;
+  gameId: {
+    id: string;
+    length: number;
+  }
+};
+
+const Game: FC<GameProps> = ({gameId, gameComplete}) =>
+  {
+    const [feedback, setFeedback] = useState<LetterMatch[][]>([]);
+    const [correct, setCorrect] = useState<boolean>(false)
+    return (
+      <div>
+        { feedback.map((guess, index) => {
+        return <Word key={index} guessLetters={guess} />
+        })
+      }
+      { !correct &&
+        <GuessInput onSubmit={ async(guess)=>{
+          const res = await fetch(`/api/game/${gameId.id}/guess`,{
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+              guessedWord: guess
+            })
+          })
+          const data = await res.json();
+          if(data.correct){
+            setCorrect(true);
+          }
+          setFeedback(() => 
+            [...feedback,
+            data.match
+            ]
+          );
+        }} length={gameId.length}/> }
+        { correct && <SubmitHighscore onSubmit={ async(name)=> {
+          const res = await fetch(`/api/game/${gameId.id}/highscore`, {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+              name: name, 
+            })
+          })
+          const data = await res.json();
+          gameComplete(data);
+        }} />}
+      </div>
+    );
+  };
+export default Game;
+
+//Helper functions
```

Comment:
What makes these components "helper functions"? To me they seem like any other component, albeit small. Maybe they deserve their own separate files, maybe they don't, that's a matter of personal preference.